docs/references/general/grid.md

```diff
@@ -0,0 +1,103 @@
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined GRID files 
+
+After GRID files are produced for seperate radars, they are sometimes further combined with other radars GRID files to meka a GRD file. Similar to GRID file just with multiple radar data included. 
+See [combine_grid](../../user_guide/make_grid.md) on how to make GRD files. 
+
+The commone naming convetion for 24-hr GRD files are:
+
+> YYYYMMDD.grd  
+
+## Fields
+
+GRID files contain a record that contains scalar and vector fields. 
+
+### Scalars
+
+| Field name       | Units      | Data Type    | Description                                                |
+| :----------      | :-----:    | :-------:    | :---                                                       |
+| *start.yearr*  | **None**   | ***short***  | Start Year of the integration period                       |
+| *start.month*  | **None**   | ***short***  | Start Month number (not padded)  of the integration period |
+| *start.day*  | **None**   | ***short***  | Start Day (not padded) of the integration period           |
+| *start.hour*  | *hrs*      | ***short***  | Start Hour of the integration period                       |
+| *start.minute*  | *min*      | ***short***  | Start Minute of the integration period                     |
+| *start.second*  | *s*        | ***short***  | Start Seconds of the integration period                    |
+| *end.yearr*    | **None**   | ***short***  | End Year of the integration period                         |
+| *end.month*    | **None**   | ***short***  | End Month number (not padded) of the integration period    |
+| *end.day*    | **None**   | ***short***  | End Day (not padded) of the integration period             |
+| *end.hour*    | *hrs*      | ***short***  | End Hour of the integration period                         |
+| *end.minute*    | *min*      | ***short***  | End Minute of the integration period                       |
+| *end.second*    | *s*        | ***short***  | End Seconds of the integration period                      |
+
+
+### Vectors 
+
+!!! Note
+    Let the number of radars in a given GRID/GRD record be defined as *numstid* and
+    Let the number of vector points in a give GRRID/GRD record be defined as *numv*. 
+
+| Field name  | Units           | Dimensionality | Data Type   | Description                                                                 |
+| :---------- | :-----:         | :-------:      | :---:       | :---                                                                        |
+| *stid*      |  **None**       |  *[numstid]*     | ***short*** | A list of of station id's that provided data for the record |
+| *channel*   |  **None**       |  *[numstid]*     | ***short*** | A list of channel numbers associated to the station id the record |
```

Comment:
Yea, I have noticed grid files will overwrite the channel number to what you set it as. So with twofsound all the channel numbers will be 0 in the grid file if you do not specify. But wasn't sure what to say in the description. 

I didn't know if grid file channels meant something else.

Reply:
I can't think of a good way to explain the channel numbers in just a few words to fit into this table. Maybe we can put an asterisk/footnote marker/"see note" here and then provide more explanation after the table? Some other fields might also benefit from some extra explanation (not necessarily in this PR)

Reply:
I agree - from a user perspective it would be helpful to have a little explanation of what it all means and to have helpful tips in there for processing (e.g. Kodiak's channel 'D' is actually what would be the primary channel for other radars etc).

Reply:
Would this issue be better addressed after the DSWG has created a naming standard? These channel naming inconsistencies ultimately arise from a lack of a standard and inconsistent treatment of channel names/numbers in the RST. 

Creating documentation will always be an iterative process. We could create a note about this in our [documentation](https://github.com/SuperDARN/rst/projects/5) project to remind us about it in the future

Reply:
I think we can make a note now and then updated once that issue is resolved. Technically users can name files to whatever they want and in the documentation, we mention common naming convention.

Reply:
Just added a note about this to the [documentation](https://github.com/SuperDARN/rst/projects/5) project. Resolving this conversation now...